stackit/internal/services/ske/project/resource.go

```diff
@@ -190,9 +190,11 @@ func (r *projectResource) Delete(ctx context.Context, req resource.DeleteRequest
 		return
 	}
 
-	if len(*clusters.Items) > 0 {
-		core.LogAndAddError(ctx, &resp.Diagnostics, "Error deleting project", fmt.Sprintln("You still have clusters in the project. Please delete them before deleting the project."))
-		return
+	if clusters != nil {
```

Comment:
you can make this into a single line
`if clusters != nil && len(*clusters.Items) > 0`